pkg/trace/stats/aggregation.go

```diff
@@ -65,8 +70,27 @@ func getStatusCode(s *pb.Span) uint32 {
 	return uint32(c)
 }
 
+// stringToBytes unsafely casts a string to a byte slice.
+func stringToBytes(str string) []byte {
```

Comment:
This should be fairly safe because:
1. We're not modifying the returned `[]byte`.
2. The only place the `[]byte` is used (read) is for the hashing function.